SDL/PixelTriplet.cuh

```diff
@@ -1,12 +1,9 @@
 #ifndef PixelTriplet_cuh
 #define PixelTriplet_cuh
 
+#include <alpaka/alpaka.hpp>
 #ifdef __CUDACC__
-#define CUDA_HOSTDEV  __host__ __device__
-#define CUDA_DEV __device__
 #else
-#define CUDA_HOSTDEV
-#define CUDA_DEV
 #endif
```

Comment:
Incomplete removal?

Reply:
Good catch, will fix in a cleanup commit now.